azext_iot/product/command_test.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.product.providers.provider import get_sdk
+from uuid import uuid4
+from knack.util import CLIError
+import os
+
+
+def initialize_workspace(cmd, product_name, working_folder="PnPCert"):
+    id = uuid4()
+    if not os.path.exists(working_folder):
+        os.mkdir(working_folder)
+
+    product_config = {
+        "id": str(id),
```

Comment:
(for later) I feel like this payload can be referenced from somewhere else, like an assets directory/module or similar